accord-core/src/main/java/accord/messages/Commit.java

```diff
@@ -174,6 +178,11 @@ public ReadNack reduce(ReadNack r1, ReadNack r2)
     @Override
     public void accept(ReadNack reply, Throwable failure)
     {
+        if (failure != null)
+        {
+            logger.error("Unhandled exception during commit", failure);
```

Comment:
should we not give this to the agent as it isn't handled?

```suggestion
            node.agent().onUncaughtException(failure); //TODO would be nice to provide "Unhandled exception during commit" for added context
```